packages/create-react-app/createReactApp.js

```diff
@@ -193,6 +193,27 @@ function createApp(
   useTypescript,
   template
 ) {
+  const unSupportedNodeVersion = !semver.satisfies(process.version, '>=8.10.0');
+  if (unSupportedNodeVersion && useTypescript) {
+    console.log(
+      chalk.yellow(
+        `You are using Node ${process.version} with the TypeScript template. Since the old unsuporoted version of tools do not supported TypeScript, you will have to upgrade.\n\n` +
```

Comment:
I would change this to `You are using Node ${process.version} with the TypeScript template. Node 8.10 or higher is required to use TypeScript.` and delete the next line.

Reply:
Done! updated the message and used `chalk.red` since this is a error message that exits the process